Hazel/src/Hazel/Core/Assert.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
```

Comment:
- This PR should resolve #326 by including `Log.h`. Also mention this in the initial post then. This PR should also close #327.
- For similar reasons we should include `Base.h` for the `HZ_STRINGIFY_MACRO`

```suggestion

#include "Hazel/Core/Base.h"
#include "Hazel/Core/Log.h"

```

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/306/commits/05c552bcb1d68cd6ffbdd6f63973fcfd8a6e408c